advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md

```diff
@@ -37,7 +37,7 @@ Conformance levels get mentioned at a couple of other points in these lessons, a
 
 ### (Before) Implementing Accessibility
 
-The lessons that follow dive into several concepts that can help you improve the accessibility of your websites, but they don't cover *every* facet of accessibility. The goal of these lessons is just to introduce you to some of the more common or simpler concepts that you should get into the habit of checking for moving forward.
+The lessons that follow dive into several concepts that can help you improve the accessibility of your websites, but they don't cover *every* facet of accessibility. The goal of these lessons are just to introduce you to some of the more common or simpler concepts that you should get into the habit of checking for moving forward.
```

Comment:
The use of "are" would be incorrect here. "Are" is a plural verb and "goal" is actually the singular subject. "Lessons" is not the subject here if that is what confused you.

In the future, try to only make these types of changes if you are certain your suggestion is correct. You can also use a service like Grammarly to double check yourself. Thanks!